packages/mux-player/src/template.ts

```diff
@@ -28,39 +23,29 @@ const isLiveOrDVR = (props: MuxTemplateProps) =>
 
 export const content = (props: MuxTemplateProps) => html`
   <${unsafeStatic(castThemeName(props.theme) ?? 'media-theme-mux')}
-    has-src="${props.hasSrc}"
-    has-open-dialog="${props.isDialogOpen}"
+    has-src="${props.hasSrc ?? false}"
```

Comment:
idea(non-blocking): 😎 eventually we may want to figure out a way to hide these requirements "under the hood" in our `html()` impl. Would also reduce redundancy/inconsistency/chance of error.